spec/fabricators/following_fabricator.rb

```diff
@@ -0,0 +1,3 @@
+Fabricator(:following) do
```

Comment:
You can also do

```ruby
Fabricator(:following) do
  user
end
```

This will give you the same result. That is shorthand for this:

```ruby
Fabricator(:following) do
  user { Fabricate(:user) }
end
```